spiketools/measures/conversions.py

```diff
@@ -61,6 +63,8 @@ def convert_train_to_times(spike_train, fs=1000):
         Spike train.
     fs : int, optional, default: 1000
```

Comment:
`fs` doesn't seem like it's optional in the code?

Reply:
Optional in the sense that it has a default value, so it's optional whether to pass in a value!